b+tree.ts

```diff
@@ -923,10 +920,9 @@ export default class BTree<K=any, V=any> implements ISortedMapF<K,V>, ISortedMap
   /** Scans the tree for signs of serious bugs (e.g. this.size doesn't match
    *  number of elements, internal nodes not caching max element properly...)
    *  Computational complexity: O(number of nodes), i.e. O(size). This method
-   *  skips the most expensive test - whether all keys are sorted - but it
-   *  does check that maxKey() of the children of internal nodes are sorted. */
+   *  validates ordering of keys (including leaves) and cached size information. */
```

Comment:
`checkValid` now does the 'most expensive check' of validating ordering? Could you make that optional?

Reply:
Sure! It's now optional, defaulting to not checking ordering.